modin/core/storage_formats/pandas/parsers.py

```diff
@@ -564,12 +564,14 @@ def update_row_nums(match):
             has_index_names=is_list_like(header) and len(header) > 1,
             skiprows=skiprows,
             usecols=usecols,
+            skip_blank_lines=False,
             **kwargs,
         )
-        # In excel if you create a row with only a border (no values), this parser will
-        # interpret that as a row of NaN values. pandas discards these values, so we
-        # also must discard these values.
-        pandas_df = parser.read().dropna(how="all")
+        pandas_df = parser.read()
+        check_nan_loc = pandas_df.isnull()[::-1].all(axis=1)
+        if check_nan_loc.all() and check_nan_loc.shape[0] > 1:
+            pandas_df = pandas_df.drop(pandas_df.index[: pandas_df.shape[0] + 1])
```

Comment:
This operation seems to leave the frame empty as it drops all of the rows, is this intended? If so, then I would suggest an easier way to create an empty frame:
```suggestion
            pandas_df = pandas.DataFrame(columns=pandas_df.columns)
```